src/TwigFakerExtension.php

```diff
@@ -40,6 +40,7 @@ public function getFunctions()
      */
     public function fakerData($type, $count = 1, $cache_key = null)
     {
+        $this->fake_data = [];
         $this->getFromCache($type, $count, $cache_key);
         if ($this->fake_data) return $this->fake_data;
```

Comment:
$this->fake_data still holds data from previous call, so it will return with old fake_data